energyplus_rmd/__main__.py

```diff
@@ -0,0 +1,4 @@
+from energyplus_rmd.runner import run
+
+
+run()
```

Comment:
Added a new `__main__` interface, so a user can run `python3 -m energyplus_rmd <args>`.  This is just an alias for running `energyplus_create_rmd` on the command line.